src/BaGet.UI/src/App.test.tsx

```diff
@@ -1,9 +1,33 @@
+// TODO: remove ' --env=jest-environment-jsdom-sixteen' from the npm test script when react-sctips get updated with newer jest version
```

Comment:
Is this the best place to keep track of the workaround?

Reply:
Yup, I think this works. I'd also create an issue on BaGet and link the URL to the issue here.